docker/init_debian.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# Note: When modifying this file, increment the Dockerfile-version minor version number
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+#
+# This is needed for mob tool to be able to pull the right image from the farm,
+# if it isn't done, bad things will happen to local builds and CI using mob tool
+# See mob tool comments for extended discussion
```

Comment:
Let's keep this warning in the dockerfile please, or in both places

Reply:
It's in both.

Reply:
ok i misread, thank you